webapp/store/views.py

```diff
@@ -89,8 +89,8 @@ def user_details(username):
         return abort(404, "User not found: {}".format(username))
 
 
-@jaasstore.route("/u/<username>/<entity_name>/")
-def user_entity(username, entity_name):
+@jaasstore.route("/u/<username>/<charm_or_bundle_name>/")
+def user_entity(username, charm_or_bundle_name):
```

Comment:
Why was this name change required? Can you add a comment to this effect as it's not clear if it was, or just to sync with the routes below.